libs/SmartSync/SmartSyncTests/MetadataManagerTests.m

```diff
@@ -44,12 +44,12 @@ @interface MetadataManagerTests : XCTestCase
 @implementation MetadataManagerTests
 
 static NSInteger const kRefreshInterval = 24 * 60 * 60 * 1000;
-static NSString* const kAccountOneId = @"001O000000m3UBiIAM";
-static NSString* const kAccountOneName = @"TEST ACCOUNT DO NOT TOUCH";
-static NSString* const kOpportunityOneId = @"006O0000006Q787IAC";
-static NSString* const kOpportunityOneName = @"TEST OPP DO NOT TOUCH";
-static NSString* const kCaseOneId = @"500O0000004Lnoa";
-static NSString* const kCaseOneName = @"00001010";
```

Comment:
These Ids no longer exist for the sdktest@cs1.com user. I had to update them to use an exisiting Case, Account , and Opportunity.